elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/AbstractHQLQueryBuilder.java

```diff
@@ -42,6 +45,7 @@
     protected static final String FROM = " FROM ";
     protected static final String JOIN = " JOIN ";
     protected static final String LEFT = " LEFT";
+    protected static final String FETCH = " FETCH ";
     protected static final String SELECT = "SELECT ";
     protected static final String AS = " AS ";
 
```

Comment:
Do we want to inline these since we've ditched codacity? I think it'd make building these join clauses clearer